script/model/forecast.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/env python3
```

Comment:
Where is this script executed? We need to add this test to the `End-to-End Self-Driving Test` stage. Probably as a separate sub test stage as the current modeling test.